easybuild/easyconfigs/n/NEST/NEST-2.12.0-foss-2015b-Python-2.7.9.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'NEST'
+version = '2.12.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.nest-simulator.org/'
+description = """NEST is a simulator for spiking neural network models
+ that focuses on the dynamics, size and structure of neural systems rather
+ than on the exact morphology of individual neurons."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/nest/nest-simulator/releases/download/v%(version)s/']
+sources = ['nest-%(version)s.tar.gz']
+checksums = ['bac578f38bb0621618ee9d5f2f1febfee60cddc000ff32e51a5f5470bb3df40d']
+
+dependencies = [
+    ('Python', '2.7.9'),
+    ('libreadline', '6.3'),
+    ('GSL', '2.1'),
+    ('libtool', '2.4.6'),
+    ('ncurses', '5.9'),
+]
+
+
+configopts = '-DREADLINE_ROOT_DIR=$EBROOTLIBREADLINE -Dwith-mpi=OFF'
+
+builddependencies = [
+    ('CMake', '3.4.1'),
+    ('Doxygen', '1.8.10'),
+    ('pkg-config', '0.27.1'),
+]
```

Comment:
@jdonners Last style-related change request: please move this up, right above `dependencies =` (same block)

Also, please include a custom `sanity_check_paths`? Usually this suffices for Python packages (unless it also includes a `/bin`):

```python
sanity_check_paths = {
    'files': [],
    'dirs': ['lib/python%(pyshortver)s/site-packages'],
}
```